.travis.yml

```diff
@@ -0,0 +1,28 @@
+Check on http://lint.travis-ci.org/ after modifying it!
```

Comment:
These line should be a comment, no? Perhaps also add a comment that this was taken from skbio?